core/modules/filter/js/filter.js

```diff
@@ -104,6 +104,21 @@ Backdrop.filterEditorDetach = function(field, format, trigger) {
   }
 };
 
+/**
+ * Provides summary text for the "Editor details" fieldset, under each CKEditor
+ * instance.
+ */
+Backdrop.behaviors.filterFieldsetSummaries = {
+  attach: function (context) {
+    $(context).find('fieldset.filter-wrapper').backdropSetSummary(function (element) {
+      var $element = $(element);
+      var editor_name = $element.find('.filter-list.form-select.filterEditors-processed :selected').text();
```

Comment:
I suspect that since this is already within the scope of `fieldset.filter-wrapper`, then `...find('select.filter-list' :selected')` should be specific enough. But havent tested.

Reply:
Thanks @docwilmot 👍 ...I am about to push another commit, in which I also address the case when a single text format is available to the current user. I have gone with `select.filterEditors-processed` and `input.filterEditors-processed` instead, which is shorter, but uses the common `.filterEditors-processed` class.